charts/prefect-server/values.yaml

```diff
@@ -293,7 +293,26 @@ ingress:
   ##           port:
   ##             name: http
 
-# Postgresql configuration
+# Secret configuration
+secret:
```

Comment:
just so i understand, `.Values.secret` is specifically for BYO postgres credentials?

Reply:
Exactly, it gives us somewhere outside of the context of `postgresql.` to define database configs (so we can dedicate `postgresql.` settings for the PostgreSQL chart without overloading it with our own chart's configs).